packages/@ionic/cli/src/definitions.ts

```diff
@@ -509,10 +537,16 @@ export interface APIResponsePageTokenMeta extends APIResponseMeta {
 
 export type HttpMethod = 'GET' | 'POST' | 'PATCH' | 'PUT' | 'DELETE' | 'PURGE' | 'HEAD' | 'OPTIONS';
 
+export enum ContentType {
```

Comment:
```suggestion
export const enum ContentType {
```

`const enums` are preferred because they compile away